packages/airnode-deployer/test/fixtures/index.ts

```diff
@@ -0,0 +1,193 @@
+import { DirectoryStructure, FileSystemType } from '../../src/utils/infrastructure';
+
+export const mockBucketDirectoryStructureList = [
+  '0xd0624E6C2C8A1DaEdE9Fa7E9C409167ed5F256c6/',
+  '0xd0624E6C2C8A1DaEdE9Fa7E9C409167ed5F256c6/dev/',
+  '0xd0624E6C2C8A1DaEdE9Fa7E9C409167ed5F256c6/dev/1662557983/',
```

Comment:
Burak was benchmarking serverless for this and serverless stores the deployment timestamp as an ISO string. It makes it more readable if you're navigating the directory manually, not sure if we'd want to do that for this PR?

Reply:
I don't really like the idea of an ISO time string in the directory name tbh. Users shouldn't access these files manually anyway. I'd suggest keeping it as it is for now. We'll add a way to fetch the files in a more user-friendly way in #1405 anyway.

Reply:
All good 👌